cmd/experimental/kjobctl/config/default/kustomization.yaml

```diff
@@ -0,0 +1,146 @@
+# Adds namespace to all resources.
+namespace: kjobctl-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kjobctl-
+
+# Labels to add to all resources and selectors.
+#labels:
+#- includeSelectors: true
+#  pairs:
+#    someName: someValue
+
+resources:
+- ../crd
+- ../rbac
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
```

Comment:
I dont think that this and the following lines are or will be relevant for kjobctl

Reply:
Removed